### ISSUE TITLE
Training time or approximation of TFLOPs?

### ISSUE THREAD

**Issue by zetian1025** [EleutherAI/pythia#87]
Hi, there seems no training time or GPU utilzation report in your paper. Can you offer us some details like approximation of TFLOPs per GPU when training the 12B model? It'll be useful to approximate training time with a similar model scale.

Thanks a lot!

**Comment by zetian1025** [comment]
Found report in Appendix D.

**Comment by haileyschoelkopf** [comment]
Yep, TFLOPs should be derivable from the train times in the paper! I can look into providing TFLOP utilization when we next update the paper.
